src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameCollector.cs

```
@@ -97,7 +99,13 @@ internal BlameCollector(IBlameReaderWriter blameReaderWriter, IProcessDumpUtilit
 
             if (this.configurationElement != null)
             {
-                this.processDumpEnabled = this.configurationElement[Constants.DumpModeKey] != null;
+                var collectDumpNode = this.configurationElement[Constants.DumpModeKey];
+                this.processDumpEnabled = collectDumpNode != null;
+                if (this.processDumpEnabled)
+                {
+                    this.collectDumpOnProcessExit = string.Equals(collectDumpNode.Attributes[Constants.CollectDumpAlwaysKey]?.Value, "true", StringComparison.OrdinalIgnoreCase);
```

Let's throw a warning for invalid argument. Bool should only be true/false

---

Also add test for when it is explicitly set to false by user.